awsx/lb/applicationLoadBalancer.ts

```diff
@@ -117,50 +117,6 @@ export class ApplicationLoadBalancer extends schema.ApplicationLoadBalancer {
       }
     }
 
-    if (listener && listeners) {
```

Comment:
This whole section was duplicated in the function. It's defined here first: https://github.com/pulumi/pulumi-awsx/blob/adae402632bd5a2b5907745ad1c3b699f2165755/awsx/lb/applicationLoadBalancer.ts#L75-L118

Luckily it didn't have a bad effect because `lbArgs.securityGroups` is set within the function.